.coveragerc

```diff
@@ -1,5 +1,7 @@
 [run]
 source = homeassistant
+relative_files = True
+branch = True
```

Comment:
But does it hurt? It produces richer coverage data and technically helps writing better tests

Reply:
If you think this is valuable for the project, please open a separate PR. This way we have one PR to discuss this instead of a PR that has a whole different scope